### ISSUE TITLE
4833 test images?

### ISSUE THREAD

**Issue by Xiaohong-Deng** [JunshengFu/semantic_segmentation#2]
Thanks for sharing. I'm refining my implementation of FCN-VGG16 using KITTI road dataset. But I don't find there are so many test images in the test set. There are just a couple hundred of them. I tried to download the dataset again from KITTI to see if they updated it. No luck.

Do you mind to elaborate on that? Or if you get them from other places, mind to share? Thanks.

**Comment by raspberryananas** [comment]
You're right, there are 290 test images. But you can use e.g. ffmpeg to break a video into a sequence of images and test your own data set. That's what I did.

**Comment by Xiaohong-Deng** [comment]
> You're right, there are 290 test images. But you can use e.g. ffmpeg to break a video into a sequence of images and test your own data set. That's what I did.

@martsalz Thanks. BTW, How do you label your data?

**Comment by raspberryananas** [comment]
That's a good question I've asked myself. Nowhere is a guide for it. This has already been discussed here:

https://stackoverflow.com/questions/47964716/how-to-format-a-data-set-for-fully-convolutional-networks

Possibly helpful:
http://www.cvlibs.net/datasets/kitti/eval_road.php

In my understanding, the labeling is only necessary for the training of own pictures.
